lowerCase.js

```diff
@@ -19,8 +19,9 @@ import words from './words.js'
  * lowerCase('__FOO_BAR__')
  * // => 'foo bar'
  */
+const rx = /['\u2019]/g
```

Comment:
@nazar-pc Just curious, how does this effect the performance? 

Im assuming that defining the regex before using it will eliminate the need for js to create a reference to it dynamically...

Reply:
Regular expressions take time to compile and additional time to execute upon some data. If you call lowerCase in a loop, regular expression will be compiler again and again on each call.

With proposed change it will be compiler once, thus each iteration will only involve executing pre-compiled regular expression, which is much faster.

Reply:
Can you rename the var to `reQuotes`.

Reply:
Done (commit amended)

Reply:
Ohhh cool thank you!